src/OpenLoco/Windows/IndustryList.cpp

```diff
@@ -637,7 +643,7 @@ namespace OpenLoco::Ui::Windows::IndustryList
             if (self->var_846 == 0xFFFF)
                 industryCost = dword_E0C39C;
 
-            if ((self->var_846 == 0xFFFF && dword_E0C39C == (1ULL << 31)) || self->var_846 != 0xFFFF)
+            if ((self->var_846 == 0xFFFF && dword_E0C39C == static_cast<currency32_t>(0x80000000)) || self->var_846 != 0xFFFF)
```

Comment:
I've been seeing this more lately. Perhaps it's time we introduced a null constant for currency32_t and 48_t? Doesn't have to happen in this PR.